assets/src/frontendIndex.js

```diff
@@ -1,3 +1,5 @@
+import 'regenerator-runtime/runtime';
```

Comment:
I'm getting this message on VS Code:

<img width="723" alt="Screenshot 2024-07-24 at 15 07 18" src="https://github.com/user-attachments/assets/f0a5abeb-5360-4fcd-a1ca-ce8d2754e5c5">

Reply:
Please, re-check it @mleray